docs/changelog.rst

```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+2.3.2 (2017-03-27)
```

Comment:
Oops sorry, let's merge this w/ the 2.3.1 section below. The reason there isn't a date is we haven't actually tagged/released it yet :)

On that subject, will you want master tagged for use immediately or should we keep accumulating fixes?